src/modules/recipies/RecipeCategories/index.jsx

```diff
@@ -0,0 +1,49 @@
+import { useNavigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { fetchCategories } from "../recipiesService";
+import "./styles.css";
+
+export const RecipeCategories = () => {
+    const [categories, setCategories] = useState([]);
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        const getCategories = async () => {
+            const categoriesData = await fetchCategories();
+            setCategories(categoriesData);
+        };
+        getCategories();
+    }, []);
+
+    const handleCategoryClick = (categoryName) => {
```

Comment:
This is not exactly as what was requested in the assignment (instead of using /recipes route with query param you should have used a dynamic route /recipes/categories/:category) but it's fine to use your approach, no need to change.